pytorch_to_returnn/torch/nn/modules/batchnorm.py

```diff
@@ -96,7 +96,7 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {
       "class": "batch_norm", "from": self._get_input_layer_name(input),
       "update_sample_only_in_training": True, "delay_sample_update": True,
-      "param_version": 1,
+      "param_version": 1, "masked_time": True,
```

Comment:
This is actually not the same as PyTorch. In PyTorch, the time is not not masked.

Reply:
You might also want to update the param version. This allows to use the efficient fused batch norm op then.

Reply:
Right. For the param version, which do you suggest?

Reply:
param_version = 2. This will change the param name (it gets the prefix "v2_") and it will not add the broadcast dims to the variable.